### DISCUSSION TITLE
Why I cannot pull version 2.14.0 stable from Artifact Hub?

### DISCUSSION THREAD

**Discussion by fbarrerafalabella** [linkerd/linkerd2#11326] [Q&A]
https://artifacthub.io/packages/helm/linkerd2/linkerd2
according o artifacthub we are on 2.11.5 version as the latest but on the repo we are on 2.14.0, do I have to pull from somewhere else?

**Comment by fbarrerafalabella** [comment]
Im using helm to install the tool btw

**Comment by alpeb** [comment]
The helm chart in 2.11 was replaced by two helm charts in 1.12:  `linkerd-crds` and `linkerd-control-plane`. You can follow the instructions [here](https://linkerd.io/2.14/tasks/install-helm/) to use them instead.

**Comment by fbarrerafalabella** [comment]
Ohhh, thats real, thanks a lot for the clarification!!
